write-api-service/src/main/java/io/ga4gh/reference/api/GitHubBuilder.java

```diff
@@ -74,22 +75,34 @@ public static void main(String[] args) {
 
     }
 
+    private void wait(String organization, String repo) {
+        int i = 0;
+        while (!repoExists(organization, repo) && i < FIVE) {
+            try {
+                TimeUnit.SECONDS.sleep(i);
+                i++;
+            } catch (InterruptedException e) {
+                e.printStackTrace();
```

Comment:
Don't stack trace and ignore, either throw a runtime exception or deal with the exception